devtools/conda-envs/test_env.yaml

```diff
@@ -4,13 +4,17 @@ dependencies:
     # Base depends
```

Comment:
I'd say we can use the same env file for both cases. Having users install pytest too doesn't make a difference and we don't have to maintain two separate envs. Rename `test_env` if you want, but if you do, remember to update the instructions in `.github/workflows`.

Reply:
Good point - I was wondering about this redundency. 

I renamed `test_env` to `env` and changed the environment name to `klifs_utils`.

I do not use GitHub Actions in this repo (yet?) but Travis, thus I updated:
`klifs_utils/.travis.yml`
`klifs_utils/devtools/README.md`

Also:
`klifs_utils/appveyor.yml`